bootstrap/gamma/simple_transfer_test.go

```diff
@@ -16,21 +16,30 @@ import (
 	"time"
 )
 
-func TestSimpleTransfer(t *testing.T) {
-	test.WithContext(func(ctx context.Context) {
-		network := NewDevelopmentNetwork(ctx, log.DefaultTestingLogger(t), "")
-		contract := callcontract.NewContractClient(network)
+func testSimpleTransfer(jsonConfig string) func(t *testing.T) {
+	return func(t *testing.T) {
+		test.WithContext(func(ctx context.Context) {
+			network := NewDevelopmentNetwork(ctx, log.DefaultTestingLogger(t), "")
```

Comment:
I think call to NewDevelopmentNetwork() should have jsonConfig as last param instead of "".